RELEASING.md

```diff
@@ -16,6 +15,38 @@ The release artifacts will be deployed into Sonatype OSS with the Maven deploy p
 You must have an account with Sonatype, and it must be associated with the JanusGraph org.
 See the relevant issue [here.](https://issues.sonatype.org/browse/OSSRH-28274)
 
+#### GPG Signing Key
+
+The JanusGraph artifacts are signed with a GPG signature.  
+If you don't already have a GPG signing key included with the `KEYS` file, you will need to create one and update the `KEYS` file. If you already have your key included into `KEYS` file then skip this step.  
+Go to  [releases](https://github.com/JanusGraph/janusgraph/releases) and download `KEYS` file (which is located in Assets) from the latest release.  
+Generate your key:
+```Shell
+gpg --full-generate-key
+```
+Select:
+*   RSA and RSA
+*   4096 bits
+*   no expiration
+*   comment: `CODE SIGNING KEY`
+
+Add your key to this file:
+```Shell
+(gpg --list-sigs <key id> && gpg --armor --export <key id>) >> KEYS.
+```
+
+After that you need to publish your key to public key servers. Sometimes those key servers are down so you may want to publish to another key servers. Choose those key servers which are required by Sonatype.  
```

Comment:
Please move each sentence onto a separate line. I think it makes sense to group the common server list here as well.

> Once your key has been created and added to the KEYS file it needs to be published to a [public key server](https://sks-keyservers.net/status/). 
> You can upload to multiple key servers within the pool or use an alternate if `pgp.mit.edu` is down.
> Sonatype typically uses the following key servers: `pool.sks-keyservers.net`, `keyserver.ubuntu.com`, `keys.gnupg.net`